index.html

```diff
@@ -4,12 +4,171 @@
+                </p>
+            </div>
+            <div>
+                <i class="balance scale icon big own-class"></i>
+                <h3>Game Features</h3>
+                <p>
+                    Get rewarded for your creativity and control with new features in the most intelligent FIFA gameplay to date
+                </p>
+            </div>
+            <div>
+                <i class="paste icon big own-class"></i>
+                <h3>Pitch Notes</h3>
+                <p>
+                    Go in-depth on the new features with all the details directly from the FIFA 21 development team.
+                </p>
+            </div>
+            <div>
+                <i class="cogs icon big own-class"></i>
+                <h3>Leagues</h3>
+                <p>
+                    FIFA 21 will allow you to play single player leagues of the real countries such as FA Cup, Premier League...
+                </p>
+            </div>
+            <div>
+                <i class="handshake icon big own-class"></i>
+                <h3>Prices</h3>
+                <p>
+                    FIFA 21 will be available for free for all subscribers of EA Play and Xbox Game Pass Ultimate.
+                </p>
+            </div>
+        </div>
+
+        <div>
+            <button class="button" type="button">Join International Fifa 2021 Conference</button>
+        </div>
+
+        <div class="message">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+
+    <section class="speakers">
+        <h2>Featured Speakers</h2>
+        <hr>
+        <div class="grid">
+            <span id="speak"></span>
+        </div>
+        
+    </section>
+
+    <div class="speakerButton">
+        <button type="button" id="showButton" class="showButton">More <i class="angle down icon large"></i></button>
+    </div>
+
+    <div class="speakerButton">
+        <button type="button" id="hideButton" class="hideButton">Less <i class="angle up icon large"></i></button>
+    </div>
+
+    <section class="partner">
+        <h2>Partners</h2>
+        <hr>
+        <div class="partners">
+            <img src="images/premier.png" alt="premier league logo" class="partner1">
+            <img src="images/laliga.png" alt="laliga logo" class="partner2">
+            <img src="images/league1.png" alt="league1 logo" class="partner3">
+            <img src="images/bundes.png" alt="bundesliga logo" class="partner4">
+            <img src="images/champions2.jpg" alt="champions league logo" class="partner6">
+            <img src="images/euro.png" alt="europa logo" class="partner6">
+        </div>
+
+    </section>
+
+    <footer class="footer">
+        <div>
+            <img src="images/unnamed.png" alt="logo">
+        </div>
+        <div>
+            <p>@2021 Kingstalux creativity<br>Some Rights Reserved</p>
+        </div>
```

Comment:
- [ ] On the mobile screen in the footer the index.html page is expected to look like the given design.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/132951919-5459fb6e-1855-4f5f-9b3d-9c0fcee3e6cc.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/132951943-04e4746e-030e-4ec8-9116-15671267ee5e.png' height='250px'> |

Reply:
Okay, I was using some initiative there but I'll remove the sections under the button.

Reply:
But I think that button is only for the speakers...I'll remove the sections under for now but if you can check with your colleagues and get back to me for clarification it'll be very nice.